packages/ses/src/module-load.js

```diff
@@ -297,6 +323,64 @@ const memoizedLoadWithErrorAnnotation = async (
   return moduleLoading;
 };
 
+function asyncOverseer() {
```

Comment:
Recommend naming asyncJobQueue or some such. At least in my dialect, overseer is best known for its use in the Bible and a troubling history of forced labor.

Reply:
I know the word as construction site term and a StarCraft2 unit :see_no_evil: 
Will fix.